src/main/java/com/cgi/ipsen5/Model/InvalidToken.java

```diff
@@ -0,0 +1,16 @@
+package com.cgi.ipsen5.Model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+
+@Setter
+@Getter
+@Entity
+@Table(name = "invalid_tokens")
+public class InvalidToken {
+    @Id
+    private String token;
```

Comment:
moet dit niet uuid zijn?

Reply:
Nee dat is de JWT token, ik sla de invalid tokens op in de database. Ik zie ook het nut niet om een (UUID) id te geven aan zo een InvalidToken row aangezien we dat toch niet gaan gebruiken, het gaat er alleen om dat de invalid tokens in de database komen te staan